catalog/ui/catalog-ui-metacard-sharing-impl/src/main/java/org/codice/ddf/catalog/ui/sharing/ShareableMetacardImpl.java

```diff
@@ -42,6 +42,8 @@
  */
 public class ShareableMetacardImpl extends MetacardImpl {
 
+  public static final String SHARING_CAPABLE_TAG = "sharing-capable";
```

Comment:
What are the cons of making this a full-fledged attribute? or rather why are we resorting to using a tag for this kind of thing.

Reply:
We can probably do that. I think we went this route since workspaces used tags a while back and the plugins checked the tags, so adding onto that was the easiest path forward. I'd almost argue that the presence of security attributes alone would be enough to imply sharing, but that may be too general.

Reply:
@pklinef Can we infer sharing through the presence of owner, groups and individuals? (SecurityAttributes)

Reply:
This tag is transient and will never be persisted. This is used within the policy plugin to then get stripped off in the policy extension plugin. We can rename is to `shareable`.